src/static/js/modules/explore-rates.js

```diff
@@ -239,91 +251,85 @@ function renderLoanAmount() {
   } else {
     params['loan-amount'] = 0;
   }
-  $('#loan-amount-result').text( formatUSD(params['loan-amount'], {decimalPlaces: 0}) );
+  $( '#loan-amount-result' ).text( formatUSD( params['loan-amount'], { decimalPlaces: 0 } ) );
 }
 
 /**
  * Render all applicable rate checker areas.
- * @return {null}
  */
 function updateView() {
   chart.startLoading();
 
   // reset view
-  dropdown(['county', 'loan-term']).hideHighlight();
+  dropdown( [ 'county', 'loan-term' ] ).hideHighlight();
 
   // Check ARM
   checkARM();
-  
 
   var data = {
-    labels: [],
-    intLabels: [],
+    labels:       [],
+    intLabels:    [],
     uniqueLabels: [],
-    vals: [],
-    totalVals: [],
-    largest: {
+    vals:         [],
+    totalVals:    [],
+    largest:      {
       label: 4,
-      val: 0
+      val:   0
     }
   };
 
   // Abort the previous request.
-  if ( typeof options['request'] === 'object' ) {
-    options['request'].abort();
+  if ( typeof options.request === 'object' ) {
+    options.request.abort();
   }
 
   // And start a new one.
-  if ( +params['loan-amount'] !== 0 ) {
-    options['request'] = getData();
-    
-    
+  if ( +Number( params['loan-amount'] ) === 0 ) {
+    resultWarning();
```

Comment:
Hmm, I'm not sure.. this update was following what the linter told me to do. I'd be inclined to merge it to get eslint in and then experiment on whether this would be needed or not. Nice find!